src/main/java/org/codehaus/mojo/taglist/TaglistReportRenderer.java

```diff
@@ -226,7 +226,19 @@ private void doCommentLine(FileReport fileReport, Integer lineNumber) {
             }
         }
 
-        tableRow(new String[] {comment, createLinkPatternedText(String.valueOf(lineNumber), link)});
+        // we can not use convenience method here ...
+        // https://github.com/mojohaus/taglist-maven-plugin/issues/165
+        sink.tableRow();
+
+        sink.tableCell();
+        text(comment);
+        sink.tableCell_();
+
+        sink.tableCell();
+        linkPatternedText(createLinkPatternedText(String.valueOf(lineNumber), link));
+        sink.tableCell_();
```

Comment:
Here you can cheat: https://github.com/apache/maven-reporting-impl/blob/a4d9b2ff9c031aa270b5366410c39bad2b0eb0ad/src/main/java/org/apache/maven/reporting/AbstractMavenReportRenderer.java#L199-L201